rmgpy/molecule/group.py

```diff
@@ -464,6 +501,26 @@ def is_specific_case_of(self, other):
                         return False
         else:
             if group.charge: return False
+        # Each site in self must have an equivalent in other
+        if self.site:
+            for site1 in self.site:
```

Comment:
I've forgotten where we spend time. Is this code worth cythoning carefully? (eg. declaring site1 site2 morphology1 morphology2 etc.) ?  (Maybe it isn't)

Reply:
I've added it to the declarations, although there isn't a cython analog to string so it just knows it's a python string. Surely this function gets called a whole lot, but typically self.site and self.morphology will be empty so those loops won't occur and those variables won't get assigned.